### ISSUE TITLE
[To-do] Better signature file descriptions and false positive warnings.

### ISSUE THREAD

**Issue by Maikuolan** [phpMussel/phpMussel#160]
*[To-do] Better signature file descriptions and false positive warnings.*

Will get around to this when I've got a spare moment (another low-priority thing which should definitely be done at some point, but there are other things I want to do first anyway; creating an issue about it so that I don't forget).

- Better descriptions for signature files are needed so that users can better decide which signature files they do/don't want/need, and so that they'll understand the purpose of each individual signature file.
- False positive risk should be ascertained and appropriate warnings provided for each signature file (this should help to address problems like the FP reported by #145).

**Comment by Maikuolan** [comment]
*Also related: #159.*

**Comment by Maikuolan** [comment]
Better descriptions, false positive warnings, etc have been implemented at the signatures repository (via https://github.com/phpMussel/Signatures/commit/167e1c242209511b2be6e99bf9cef67d71c780f0 and https://github.com/phpMussel/Signatures/commit/eee54d8c4ed14c6678e3524c07693a49eb5d29a8). This information will automatically display to users when they visit that repository, and when they access the front-end updates page. That should be sufficient.

I've removed the L10N label from this issue, because after writing that information, I can see that translating it all would effectively double the size of all localisations, adding quite a bit of unwanted bloat, and would also be difficult to maintain. Thinking I'll skip translating this newly written information (won't affect the docs or core in any way anyway).

Basically, this is resolved now. Closing.
